test/distributed/checkpoint/test_traverse.py

```diff
@@ -11,18 +11,20 @@
 if TYPE_CHECKING:
     from torch.distributed.checkpoint.metadata import STATE_DICT_TYPE
 
+from torch.testing._internal.common_device_type import instantiate_device_type_tests
+
 
 class TestTraverse(TestCase):
     """
     Test class for util methods of _traverse
     """
 
-    def test_traverse_shallow(self) -> None:
+    def test_traverse_shallow(self, device) -> None:
```

Comment:
@fegin please comment on whether these tests need to be extended to other devices than `cpu`.  If not we can keep current test unchanged.

Reply:
The change to this file, test_traverse.py, is NOT necessary. This test verified only the CPU logic, how to traverse the keys. Where do the tensors locate shouldn't matter.